backend/src/utils/notFound.ts

```diff
@@ -0,0 +1,14 @@
+import { Response } from 'express';
+
+/**
+ *
+ * @param response respone object
+ * @param resource the resource not found - optional
+ */
+function notFound(response: Response, resource = 'resource') {
```

Comment:
be consistent in es6:
`const notFound = (...){
  ...
}`

Reply:
Also you don't need a util function for this, express should handle not found routes in the app level. You can do a research on that. This should give you a hint: https://levelup.gitconnected.com/how-to-handle-errors-in-an-express-and-node-js-app-cb4fe2907ed9

Reply:
The util is not for handling not found routes but for handling resources not found.

There's already an app level middleware for handling not found routes.